.github/workflows/test-signer-linux.yml

```diff
@@ -13,6 +13,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Bootstrap SoftHSM Credentials
+      run: ./scripts/sofths_setup.sh
```

Comment:
You will need to set the executable bit on your file to run it like this.

`$ man chmod` to learn how to do this.

Reply:
I ran the command "$ sudo chmod +x ./scripts/softhsm_setup.sh" after talking with Angela.

I think there's a new error now with the function declaration on line 23 of the script.

Reply:
I don't have any issues on my cloudtop, does it work on yours?

Reply:
Mine also ran without issue

Reply:
I tried, and I got the following error: https://paste.googleplex.com/6685965933346816

Reply:
That's because the script is not idempotent, which is okay for a CI perspective